sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -950,8 +950,18 @@ trait NullAwareHashedRelation extends HashedRelation with Externalizable {
 
 /**
  * A special HashedRelation indicates it built from a empty input:Iterator[InternalRow].
+ * get & getValue will return null just like
```

Comment:
BTW where do we call these methods for `EmptyHashedRelation`?

Reply:
yes. It will be called. EmptyHashedRelation now is also applied at all joinType. I confirmed it while running UT.